src/index.html

```diff
@@ -9,10 +13,43 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.scss"
```

Comment:
The `href` attribute is linking to a SCSS file (`styles/main.scss`). Browsers cannot directly interpret SCSS files. You need to compile the SCSS file into a CSS file and link to the resulting CSS file instead.